src/core/components/AMInstallButton/index.js

```diff
@@ -86,6 +95,7 @@ type ButtonProps = {|
 |};
 
 const TRANSITION_TIMEOUT = 150;
+export const NOT_IN_EXPERIMENT = 'notInExperiment';
```

Comment:
Maybe this should also be listed in `amo/components/InstallWarning` so that all variables/values for this experiment are in the same component.